traefik/values.yaml

```diff
@@ -323,9 +355,18 @@ ports:
 #       - CurveP384
 tlsOptions: {}
 
-# Options for the main traefik service, where the entrypoints traffic comes
-# from.
+# Options for the traefik services, where the entrypoints traffic comes from.
```

Comment:
Options for the Traefik services, where the entrypoints traffic comes from.

Reply:
ok